src/TinyMCEInput.js

```diff
@@ -82,7 +82,8 @@ var TinyMCEInput = React.createClass({
       pollInterval: 1000,
       textareaProps: {},
       otherEventHandlers: {},
-      onChange: function() {}
+      onChange: function() {},
+      component: 'textinput',
```

Comment:
I tried this on a both Windows and Mac, with Chrome, FF, and IE. It works everywhere. Which OS/browser are you using? Also, which version of React. I just checked both of my environments, and they are 15.5+